docs/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+AdvancedMH = "5b7e9947-ddc0-4b3f-9b55-0d8042f74170"
```

Comment:
I usually don't add it to docs/Project.toml since the GH action adds it anyway and it creates problems with CompatHelper.
```suggestion
```